frontend/public/components/sidebars/build-config-sidebar.jsx

```diff
@@ -95,14 +95,14 @@ spec:
 
 const samples = [
   {
-    header: 'Docker build',
-    details: 'The Docker build strategy invokes the docker build command, and it therefore expects a repository with a Dockerfile and all required artifacts in it to produce a runnable image.',
+    header: 'Build from Dockerfile',
+    details: 'A Dockerfile build runs an image build using a Dockerfile in the source repository or specified in the build config itself.',
```

Comment:
s/runs/performs/  ("runs" is easy to confuse w/ the idea of running a container).